sig-contributor-experience/annual-report-2021.md

```diff
@@ -4,79 +4,105 @@
+  | Events                    | Office Hours                           |
+  | Events                    | Elections                              |
+  | GitHub Management         | GitHub Admin / Moderation              |
+  | GitHub Management         | GitHub Master -> Main rename           |
+  | GitHub Management         | GitHub New Membership Coordinator      |
+  | Mentoring                 | Google Summer of Code                  |
+  | Mentoring                 | Group Mentoring                        |
+  | Mentoring                 | LFX Mentor Program                     |
+  | Mentoring                 | Meet our Contributors                  |
+  | Mentoring                 | Outreachy                              |
+  | Slack Infra               | slack-infra                            |
 
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
-
-   -
-   -
-   -
+   
+   - GitHub Admin Subproject
+      - Looking to bring on more membership co-ordinators 
+   - Community Management Automation
+      - [Zoom to Youtube](https://github.com/kubernetes/community/issues/5201)
+      - [Workspace Automation](https://github.com/kubernetes/steering/issues/213)
+         - Mailing list management
+         - Calendar management
+         - Shared drives
+      - [Mailing list migration](https://github.com/kubernetes/community/issues/6353)
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Issue Velocity
+      - We are a service orientated sig, Issues are the main form of how we track work.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - We don't have a CONTRIBUTING.md as our SIG's contributing proccess does not differ from the proccess in the Contributor Guide.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - N/A
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes. [25+ Different groups](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Contributor%20Experience&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1577854800000&to=1640926800000&viewPanel=1)
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - More full time support for the items under bullet point 1.
```

Comment:
A full time community manager who can maintain several subprojects and help with the overall direction of the group